src/components/CardListItem.js

```diff
@@ -23,7 +23,7 @@ const renderCardContent = (item, horizontal) => {
         <Image
           source={{ uri: picture.url }}
           style={stylesWithProps({ horizontal }).image}
-          containerStyle={(styles.imageContainer, !!imageStyle && imageStyle)}
+          containerStyle={[styles.imageContainer, !!imageStyle && imageStyle]}
```

Comment:
`containerStyle={[styles.imageContainer, imageStyle]}` you do not event have to check for the `imageStyle` in the array. if it is not defined, `containerStyle={[{...}, undefined]}` works totally fine

Reply:
Yes, you are right. I think the same applies to the other components. I'll fix them all.

`<RegularText
          small
          style={[!!generalStyle && generalStyle, !!overtitleStyle && overtitleStyle]}
        >` -> `<RegularText small style={[generalStyle, overtitleStyle]}>`

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/4550329f16ccdef1de562c18775b5f594b9564b4